docs/how-to-guides/networkx.md

```diff
@@ -5,10 +5,134 @@ sidebar_label: Utilize the NetworkX library
+
+<img src={require('../data/how-to-guides/nxalg-how-to-3.png').default} className={"imgBorder"}/>
+
+Copy the query, go to the **Query Execution** tab and paste the query into the **Cypher Editor**. If you need the query, here it is:
+
+```cypher 
+CALL nxalg.is_bipartite() YIELD is_bipartite;
+```
+
+By clicking on the **Run Query** button, you can see that the Europe backpacking graph is not bipartite.
+
+<img src={require('../data/how-to-guides/nxalg-how-to-4.png').default} className={"imgBorder"}/>
+
+In the same way, you can run other procedures from the `nxalg` module and the procedures from the `graph_analyzer` module, which can be found in the **Query Modules** section.
+
+## How to implement custom NetworkX module
+
+Besides using already implemented modules, you can create your own module which utilizes the NetworkX library.
+To learn how to implement a custom query module, head over to the [example of query module in Python](/docs/memgraph/reference-guide/query-modules/implement-custom-query-modules/custom-query-module-example#python-api).
+
+Since Memgraph is integrated with NetworkX, you can import NetworkX library inside your Python code and start coding. In this guide, we will create a new query module which utilizes the NetworkX library with the help of Memgraph Lab.
+
+### 1. Connect to Memgraph
+
+To be able to try out this example on your own, you need a running Memgraph instance. Run Memgraph using the Memgraph Platform Docker image, which includes both the MAGE library and Memgraph Lab, a visual interface. 
+To run the image, open a command-line interpreter and run the following Docker command:
+
+```
+docker run -it -p 7687:7687 -p 7444:7444 -p 3000:3000 memgraph/memgraph-platform:latest
+```
+
+[Connect to Memgraph](/docs/memgraph-lab/connect-to-memgraph#connecting-to-memgraph) via Memgraph Lab which is running at `localhost:3000`. 
+
+Check out the [installation guide](/docs/memgraph/installation) for other installation options. If you wish to avoid the installation, you can also use [Memgraph Cloud](/docs/memgraph-cloud/).
+
+### 2. Load the dataset
+
+First, load a dataset in the **Datasets** section by searching and loading the Karate club friendship network dataset. 
+
+<img src={require('../data/how-to-guides/nxalg-how-to-5.png').default} className={"imgBorder"}/>
+
+### 3. Implement a custom query module
+
+Once the dataset is loaded, go to the **Query modules** section. The goal is to create a community detection algorithm that can partition a network into multiple communities with the help of the NetworkX library. Click on the **New Module** and type in the module name, e.g., `communities`. 
+
+<img src={require('../data/how-to-guides/nxalg-how-to-6.png').default} className={"imgBorder"}/>
+
+There is a sample Python code on the next screen, inside the code editor. Select it, delete it and paste the following code:
+
+```python
```

Comment:
You don't want to explain the code a bit? Just in a few sentences, or make comments in the code?

Reply:
No. Just kidding. I will, thanks

Reply:
No is a perfectly fine answer to me if you think it should be evident from the code itself :D